website/scripts/build-merged-schema.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+
+
+  friendlyName: 'Build merged schema',
```

Comment:
Nitpick: In Sails world, "Generate" is a more common way to explain a script like this that generates a file